soundcraft/notepad.py

```diff
@@ -56,7 +49,7 @@ def __init__(
             except Exception:
                 # Fall-back to class name, since reading the product over USB requires write access
                 self.product = self.__class__.__name__
-            self.fwVersion = f"{major}.{minor}"
+            self.fwVersion = "%d.%02d" % (major, minor)
```

Comment:
We can still use f-strings for this:

```
self.fwVersion = f"{major}.{minor:02d}"
```

Reply:
Some things are both less than obvious and less than well documented :-/

Anyways, do you
  * want me to force-push a changed commit which uses an f string
  * want me to add another commit which fixes this to use an f string
  * want to fix it yourself
?

Reply:
Sorry, `pre-commit` running `pytest` still produces this:

```
>           self.fwVersion = f"{major}.{minor:02d}"
E           TypeError: unsupported format string passed to MagicMock.__format__

soundcraft/notepad.py:65: TypeError
```

Reply:
Agreed.  I'm happy with this as-is then :)